### PR TITLE
DD-15: finished updating the HTCondor playbook.

### PR DESCRIPTION
**Pull request by slr71**
I had to make some pretty significant changes to get HTCondor to work when I did the RFHS installation, so I updated these deployment playbooks to use a configuration similar to the one at RFHS. This may remove some important configuration settings, however.

One important change is that the file, `condor_config.local`, is no longer used. Instead, this playbook generates multiple files under `/etc/condor/config.d`.

### CODE REVIEW

File: ansible/condor/install.yml [hunk 1]
```diff
@@ -39,9 +39,32 @@
...
+        config_path: "~condor/.docker/config.json"
+      loop: "{{docker_registries}}"
+      when: docker_registries is defined
+
+    - ansible.builtin.file:
+        path: "~condor/.docker"
+        owner: "condor"
+        group: "condor"
+        recurse: true
+        state: "directory"
```
Comment (slr71) [inline ansible/condor/install.yml:70]: I was getting errors when I tried to set `become_user` to `condor` before logging in, so I simply specified the path to the configuration file then fixed the permissions after.

File: ansible/condor/roles/de-image-janitor/templates/excludes.j2 [hunk 2]
```diff
@@ -1,24 +1 @@
...
-cyverse/jupyterlab-nanodj:latest
-{{ docker.registry.base }}/jupyterlabqiime2:2018.11.2
-{{ docker.registry.base }}/genomicseducationalliance-jupyterlab-rnaseq-qc:dev_2.1
-{{ docker.registry.base }}/quaternionclassifier:latest
-{{ docker.registry.base }}/powerstateclassifier:latest
-{{ docker.registry.base }}/behaviorclassifier:latest
-{{ docker.registry.base }}/turboprop:latest
-genomicseducationalliance/rna-seq-leptin:dev_0.2
-cyversevice/rstudio-geospatial:terraref-3.6.0
+{{porklock_image}}:{{porklock_tag}}
```
Comment (slr71) [inline ansible/condor/roles/de-image-janitor/templates/excludes.j2:1]: Most of the old exclusions were outdated, so I removed all of them except for this one.

Comment (ianmcorvidae) [reply ansible/condor/roles/de-image-janitor/templates/excludes.j2:1]: The only other one I could think of would be the url-import tool, but that'll also be quite small and I dunno how much it's used. So porklock is probably sufficient.

**Comment by ianmcorvidae** [comment]
Looking at both this and the folder in your branch, I don't spot anything, so I'd say :shipit:. We should definitely run some stuff for testing once deployed, since we've switched things over to using the roles system vs. our existing QA/Prod deployments, but all that is much, much cleaner, and I'm guessing it's been in use for something external already.

File: ansible/condor/roles/de-image-janitor/templates/excludes.j2 (hunk 2 above)
Comment (slr71) [reply ansible/condor/roles/de-image-janitor/templates/excludes.j2:1]: That's a good point. I'll add it to the list.

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**